src/Juvix/Compiler/Backend/Html/Extra.hs

```diff
@@ -51,6 +51,12 @@ jsLink js = do
       ! Attr.type_ "text/javascript"
     $ mempty
 
+closeAll :: Html
+closeAll =
+  script
+    ! Attr.type_ "text/javascript"
+    $ "function closeAll() { document.body.querySelectorAll('summary').forEach((e) => { document.body.querySelectorAll('details').forEach((e) => (e.hasAttribute('open')) ? e.removeAttribute('open') : '')})}"
```

Comment:
do you think we should have another button to expand everything? or maybe use the same button?
Either way, I think it would be nice to be able to do that